Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/KeyboardInputFieldBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.UI;
+#if WINDOWS_UWP
```

Comment:
why can't we define these for non-UWP builds?

Reply:
I wanted those scripts to work with the UWP keyboard only. Isn't it right that for other platforms InputFields are capable of starting up native keyboards themselves?